libs/cli/langgraph_cli/analytics.py

```diff
@@ -0,0 +1,72 @@
+import functools
+import os
+import pathlib
+import platform
+from typing import Any
+
+import httpx
+from dotenv import load_dotenv
+
+from langgraph_cli.constants import DEFAULT_CONFIG, DEFAULT_PORT
+from langgraph_cli.version import __version__
+
+load_dotenv()
+
+
+def get_anonymized_params(kwargs: dict[str, Any]):
+    params = {}
+
+    # anonymize params with values
+    if "config" in kwargs:
+        params["config"] = kwargs["config"] != pathlib.Path(DEFAULT_CONFIG).resolve()
+
+    if kwargs.get("docker_compose"):
+        params["docker_compose"] = True
+
+    if "port" in kwargs:
+        params["port"] = kwargs["port"] != DEFAULT_PORT
+
+    if kwargs.get("debugger_port"):
+        params["debugger_port"] = True
+
+    if kwargs.get("postgres_uri"):
+        params["postgres_uri"] = True
+
+    # pick up exact values for boolean flags
+    for boolean_param in ["recreate", "pull", "watch", "wait", "verbose"]:
+        if kwargs.get(boolean_param):
```

Comment:
Should this have "is not None"

Reply:
depends on whether we actually want to log `False` for booleans -- I also considered in the beginning but decided against it. we can just always assume that if the flag is not in the `params` it's set to `False`